src/coreclr/src/pal/src/exception/seh-unwind.cpp

```diff
@@ -111,6 +111,22 @@ static void WinContextToUnwindContext(CONTEXT *winContext, unw_context_t *unwCon
     unwContext->regs[13] = winContext->Sp;
     unwContext->regs[14] = winContext->Lr;
     unwContext->regs[15] = winContext->Pc;
+#elif defined(HOST_ARM64)
```

Comment:
@sdmaclea, was it not required in the previous version of libunwind?

Reply:
Between 1.3rc1 and 1.5rc2, libunwind changed the definition of `unw_context_t`.  Therefore `#if UNWIND_CONTEXT_IS_UCONTEXT_T` changed from being evaluated as `true` to `false`.  Now this code is used.  

This is functionally identical to the code above.
```c++
static void WinContextToUnwindContext(CONTEXT *winContext, unw_context_t *unwContext)
{
#define ASSIGN_REG(reg) MCREG_##reg(unwContext->uc_mcontext) = winContext->reg;
    ASSIGN_UNWIND_REGS
#undef ASSIGN_REG
}
```